### PR TITLE
[v2.9] KDM OOB Release

### PR DESCRIPTION
**Pull request by kinarashah**
https://github.com/rancher/rancher/issues?q=is%3Aopen+is%3Aissue+milestone%3Av2.9-KDM-Jul-2024-RKE1-patches

### CODE REVIEW

File: pkg/rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -10352,6 +10352,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
+		// Enabled in Rancher v2.9.1
```
Comment (jiaqiluo) [inline pkg/rke/k8s_rke_system_images.go:10355]: just to mention that those versions are actually released in the OOB release.

**Review by jiaqiluo — APPROVED** [review]

File: pkg/rke/k8s_rke_system_images.go (hunk 1 above)
Comment (kinarashah) [reply pkg/rke/k8s_rke_system_images.go:10355]: Ah oops, Sergey mentioned it earlier but looks like infracloud is yet to update it, @krunalhinguu could you fix this with a later PR?

Comment (kinarashah) [reply pkg/rke/k8s_rke_system_images.go:10355]: Won't delay release for this as no changes in data.json

Comment (jiaqiluo) [reply pkg/rke/k8s_rke_system_images.go:10355]: @kinarashah @krunalhinguu , please fix the same problem on the 2.8 branch https://github.com/rancher/kontainer-driver-metadata/pull/1457

**Pull request merged**